backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepositoryTest.java

```diff
@@ -801,6 +802,17 @@ void updateUserPrivileges_illegalGraphqlArgumentException_whenGroupToAddNotFound
     when(mockGroupProfile.getName()).thenReturn(groupOrgDefaultName);
     when(_client.listGroups(isNull(), eq("profile.name sw \"" + groupOrgPrefix + "\""), isNull()))
         .thenReturn(mockGroupList);
+
+    Set<Facility> userFacilities = Set.of();
+    Set<OrganizationRole> userOrgRoles = Set.of(OrganizationRole.USER);
+
+    try {
+      _repo.updateUserPrivileges(userName, org, userFacilities, userOrgRoles);
+      Assert.fail("Expected an IllegalGraphqlArgumentException to be thrown");
+    } catch (IllegalGraphqlArgumentException e) {
+      assertEquals(
+          "Cannot add Okta user to nonexistent group=" + groupOrgPrefix + ":USER", e.getMessage());
+    }
```

Comment:
Could this be replaced with [assertThrows](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepositoryTest.java#L769-L777)?